src/components/new-request/search.vue

```diff
@@ -742,6 +751,20 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
         this.setEntityTypeCd(null)
       }
     }
+
+    if (this.isChangeName) {
+      if (this.isBenBcUlcCccCorpType) {
+        this.setLocation(Location.BC)
+        this.setEntityTypeCd(this.corpTypeToEntityType(this.business?.legalType as unknown as CorpTypeCd))
+      } else {
+        this.setEntityTypeCd(this.business.legalType)
+      }
+
+      if (this.isChangeNameXpro) {
+        this.setLocation(Location.CA)
```

Comment:
What about international xpro?

Reply:
Jurisdiction is not known when a search is performed in Business Lookup. Only Legal Type is returned. No jurisdiction details

Reply:
Oh, you're right -- the business might be xpro but it's registered in BC.

This is fine. Resolving.